nixos/lib/testing-python.nix

```diff
@@ -0,0 +1,266 @@
+{ system
+, pkgs ? import ../.. { inherit system config; }
+  # Use a minimal kernel?
+, minimal ? false
+  # Ignored
+, config ? {}
+  # Modules to add to each VM
+, extraConfigurations ? [] }:
+
+with import ./build-vms.nix { inherit system pkgs minimal extraConfigurations; };
+with pkgs;
+
+let
+  jquery-ui = callPackage ./testing/jquery-ui.nix { };
+  jquery = callPackage ./testing/jquery.nix { };
+
+in rec {
+
+  inherit pkgs;
+
+
+  testDriver = stdenv.mkDerivation {
+    name = "nixos-test-driver";
+
+    buildInputs = [ makeWrapper python3Full ];
+
+    dontUnpack = true;
+
+    preferLocalBuild = true;
+
```

Comment:
Could there be a `checkPhase` that runs `pylint --errors-only` and `black --check` to verify logically as much as possible and that we have a common style (without style discussions)
https://github.com/psf/black

Reply:
oh `black` is nice... this could also be run over *all* test scripts.